source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -284,6 +283,22 @@ void MaterialXShadingNodeImpl<BASE>::updateShader(MHWRender::MShaderInstance& sh
         udimIdentifiers = &(udimSetValue->asA<mx::StringVec>());
     }
 
+    // Address mode mapping
+    const std::vector<MSamplerState::TextureAddress> addressModes
+    {
+        MSamplerState::TextureAddress::kTexBorder,
```

Comment:
It's a bit concerning that these vectors need to be implicitly kept in sync with the MaterialX enums.

Reply:
Maybe a map is preferable?

Reply:
Yes, I think it would be better.